### PR TITLE
(SIMP-4415) k8s: Update readme to describe master

### PR DESCRIPTION
**Pull request by jeefberkey**
- Add section to README describing how to set up a master
  - Only one is supported at this time
- Update CI yamls

### CODE REVIEW

**Comment by jeefberkey** [comment]
✅ https://gitlab.com/simp/pupmod-simp-simp_kubernetes/pipelines/17496376

**Comment by jeefberkey** [comment]
[pipeline status](https://gitlab.com/simp/pupmod-simp-simp_kubernetes/commits/SIMP-4415-readme)

File: README.md [hunk 1]
```diff
@@ -86,12 +87,39 @@ simp_kubernetes only sets up the services listed above, with a few catches:
...
 ```puppet
 class { 'simp_kubernetes':
   etcd_peers   => ['kube-master01.domain.net'],
   kube_masters => ['kube-master01.domain.net','kube-master02.domain.net','kube-master03.domain.net']
 }
 ```
+or in hiera:
+```yaml
+---
+simp_kubernetes::is_master: true
```
Comment (lnemsick-simp) [inline README.md]: shouldn't this be false in general and only true on the master?

File: README.md [hunk 2]
```diff
@@ -86,12 +87,39 @@ simp_kubernetes only sets up the services listed above, with a few catches:
...
+  etcd_peers   => ['kube-master01.domain.net'],
+  kube_masters => ['kube-master01.domain.net','kube-master02.domain.net','kube-master03.domain.net'],
+  is_master    => true
+}
+```
+or in hiera:
+```yaml
+---
+simp_kubernetes::is_master: true
+```
```
Comment (lnemsick-simp) [inline README.md]: this assumes hiera above from all nodes?

File: README.md (hunk 1 above)
Comment (jeefberkey) [reply README.md]: Yes! Whoops

File: README.md (hunk 2 above)
Comment (jeefberkey) [reply README.md]: I hoped to clear this up. I don't want to assume how a hierarchy is set

File: README.md [hunk 3]
```diff
@@ -16,10 +16,11 @@
 
 ## Description
 
-**FIXME:** Ensure the *Description* section is correct and complete, then remove this message!
-
 Minimally manage a Kubernetes cluster! This module will configure the core
-services required to get a basic Kubernetes cluster up and running.
+services required to get a basic Kubernetes cluster up and running, based on the
+RedHat distribution available in the `extras` CentOS repo.
```
Comment (trevor-vaughan) [inline README.md]: This doesn't parse for me. You should probably say something like 'based on the Red Hat provided packages'. Mixing RHEL and CentOS will confuse people.

**Review by lnemsick-simp — APPROVED** [review]

**Pull request merged**